tests/e2e/specs/api/BuildPushRunPodmanContainerAPI.spec.ts

```diff
@@ -45,19 +45,10 @@ export TKN=$(oc whoami -t)
 export REG="image-registry.openshift-image-registry.svc:5000"
 export PROJECT=$(oc project -q)
 export IMG="\${REG}/\${PROJECT}/hello:\${DATE}"
-
-# Create test directory and Dockerfile
-mkdir -p /projects/dockerfile-test
-cd /projects/dockerfile-test
-
-cat > Dockerfile << EOF
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
-RUN echo "Hello from Kubedock!" > /hello.txt
-CMD ["cat", "/hello.txt"]
-EOF
+cd $PROJECT_SOURCE
 
 podman login --tls-verify=false --username "\${USER}" --password "\${TKN}" "\${REG}"
-podman build -t "\${IMG}" .
+podman build -t "\${IMG}" -f Dockerfile.\${ARCH} .
```

Comment:
It could be helpful if you could leave a comment in the code about what should should be inside  `Dockerfile` file in the root folder of test repo to make this test working.